packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -219,7 +220,6 @@ function RevisionsButtons( {
 								</p>
 							) : (
 								<Button
-									disabled={ areStylesEqual }
```

Comment:
Due to the rendering logic, this button will never be shown when `areStylesEqual` is truthy. (⚠️ Please double-check that I'm reading this correctly!)